static/js/settings_account.js

```diff
@@ -127,23 +130,21 @@ exports.add_custom_profile_fields_to_settings = function () {
 
             function update_custom_user_field() {
                 var fields = [];
-                var user_id = user_pill.get_user_ids(pills);
-                if (user_id.length > 1) {
-                    ui_report.message(i18n.t("Only one user allowed"), $("#custom-field-status"), 'alert-error');
-                    return;
-                }
-                if (user_id.length < 1) {
+                var user_ids = user_pill.get_user_ids(pills);
+                if (user_ids.length < 1) {
                     fields.push(field.id);
                     update_user_custom_profile_fields(fields, channel.del);
                 } else {
-                    fields.push({id: field.id, value: user_id[0]});
+                    fields.push({id: field.id, value: user_ids});
                     update_user_custom_profile_fields(fields, channel.patch);
                 }
             }
 
             if (value) {
```

Comment:
This should be a check for `            if (value !== undefined && value.length > 0) {                                           ` 

(Otherwise, if you reload the page after deleting the last user, it doesn't work).